### ISSUE TITLE
link sticker

### ISSUE THREAD

**Issue by mahdi-saberi** [adw0rd/instagrapi#463]
as you know link sticker recently added to instagram, and now its public; so every body can use! would you please add it to project?

**Comment by adw0rd** [comment]
### Compensate with donations ($30 and more) for the time I spent to implement new functionality

[![Donate](https://www.buymeacoffee.com/assets/img/custom_images/yellow_img.png)](https://www.buymeacoffee.com/adw0rd)

* Bitcoin: 3KHgnvNNWio6wyQ71cgNeBQojXStxzkzfd
* Ethereum: 0x6e644c6b975B47801072b07a9a57DDfEC10DE74F
* https://paypal.me/adw0rd
* For Russia: СБП, Qiwi и тп: **+79118107609**

**Comment by adw0rd** [comment]
Implemented in 1.16.6

**Comment by adw0rd** [comment]
Photo:

```python
>>> from instagrapi import Client
>>> from instagrapi.types import StoryLink
>>> from instagrapi.story import StoryBuilder

>>> cl = Client(proxy='http://123.123.123.123:40007')
>>> cl.load_settings('/tmp/dump.json')

>>> cl.photo_upload_to_story("/tmp/tmpg133wsw2.mp4.jpg", "", links=[StoryLink(webUri="http://adw0rd.com")])
```

Video:

```python
>>> from instagrapi import Client
>>> from instagrapi.types import StoryLink
>>> from instagrapi.story import StoryBuilder

>>> cl = Client(proxy='http://123.123.123.123:40007')
>>> cl.load_settings('/tmp/dump.json')

>>> link = StoryLink(webUri="http://adw0rd.com")
>>> buildout = StoryBuilder("/tmp/tmptsktx_j6.mp4", "Test").video(5, link=link.webUri)
Moviepy - Building video /tmp/tmpxf3x57e8.mp4.
Moviepy - Writing video /tmp/tmpxf3x57e8.mp4

Moviepy - Done !
Moviepy - video ready /tmp/tmpxf3x57e8.mp4

>>>  cl.video_upload_to_story(buildout.path, "", stickers=buildout.stickers)
Analizing video file "/tmp/tmpxf3x57e8.mp4"
Generating thumbnail "/tmp/tmpxf3x57e8.mp4.jpg"...

Story(pk='2737326000305270904', id='2737326000305270904_29817608135', code='CX8736KovB4', ....
```
